.github/workflows/codeql-analysis.yml

```diff
@@ -4,6 +4,7 @@ on:
   pull_request:
     paths:
       - "**.go"
+    types: ready_for_review
```

Comment:
If we do not go via draft, would that work?

Reply:
then it should run no matter what

Reply:
From the doc, it says this event is sent when a PR goes from draft to R4R (https://docs.github.com/en/webhooks-and-events/events/issue-event-types#ready_for_review).
So if I open my PR directly in as R4R will it work as well?